PinpointKit/PinpointKit/Sources/Fonts.swift

```diff
@@ -32,7 +32,7 @@ public extension UIFont {
      
      - returns: A Source Sans Pro `UIFont` at the specified size and weight.
      */
-    public static func sourceSansProFontOfSize(_ fontSize: CGFloat, weight: FontWeight = .regular) -> UIFont {
+    public static func sourceSansProFont(ofSize fontSize: CGFloat, weight: FontWeight = .regular) -> UIFont {
```

Comment:
And good point about `system`. I go back and forth on `UIFont.system`, personally. In context, I think it would seem reasonable on its own: `label.font = .system`, but with a parameter, not so much: `label.font = .system(ofSize: 32)`.